### PR TITLE
Add github logo with link to this repo in the footer

### PR DESCRIPTION
**Pull request by kharann**
Title says it all bois. Relevant issue #105

### CODE REVIEW

**Comment by kharann** [comment]
Jif
git

**Review by FredrikAugust — APPROVED** [review]

**Comment by FredrikAugust** [comment]
LGTM

**Comment by plusk** [comment]
Do an opacity reduction for the hover instead of making it black, the contrast isn't super great on the blue
Good work either way

**Comment by kharann** [comment]
@plusk Does this look better b0ss
git2

File: src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  @media #{$tabletAndUp} {
+    margin-top: $vertical-space-large;
+  }
+  svg {
+    margin-left: 0.5rem;
+    fill: #ffffff;
+    transition: filter 0.4s;
+
+    &:hover {
+      filter: brightness(0.4);
```
Comment (plusk) [inline src/components/Footer.vue]: Drop the filter

File: src/components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      opacity: 1;
```
Comment (plusk) [inline src/components/Footer.vue]: Turn this value down on hover

File: src/components/Footer.vue (hunk 1 above)
Comment (kharann) [reply src/components/Footer.vue]: ayay captian, i did like the animation tho :((

**Comment by kharann** [comment]
git3

**Pull request merged**